amy/templates/forms/trainingrequest.html

```diff
@@ -8,52 +8,19 @@
 
 {% block content %}
 
-<p class="lead">
-  Our instructor training classes are for people who wish to learn how to
-  teach Carpentry workshops. We teach impactful pedagogical techniques and
-  prepare you to become an Instructor in our community. We use this form to
-  register interest and build new communities of Instructors around the world.
+<p>
+  This form serves as the entry point for Instructor Training for two groups:
```

Comment:
You removed `class="lead"`, which rendered
> Our instructor training classes are for people who wish to learn how to...

in slightly bigger font. Are you sure you don't want users to highlight on this first paragraph?

Reply:
Correct - since we have changed the content of this text, we no longer need a section highlighted.